src/EditorFeatures/Test/SymbolFinder/DependentTypeFinderTests.cs

```diff
@@ -4,19 +4,58 @@
 
```

Comment:
renaming the file made this show as two different files, so i'm going to rename later.